src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,31 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
+import mate.academy.hibernate.relations.exceptions.DataNotFoundException;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import org.hibernate.SessionFactory;
+
+public class MovieServiceImpl extends AbstractService implements MovieService {
+    private final MovieDao movieDao;
+
+    public MovieServiceImpl(SessionFactory sessionFactory) {
+        super();
+        this.movieDao = new MovieDaoImpl(sessionFactory);
```

Comment:
Consider using dependency injection for the `MovieDao` instead of instantiating it directly in the constructor. This will improve the flexibility and testability of the code.